src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2804,7 +2780,7 @@ var projectUniqueNamesForBuildIntToUpdate
                 var referenceContext = new DependencyGraphCacheContext(logger, Settings);
 
                 var now = DateTime.UtcNow;
-                Action<SourceCacheContext> cacheContextModifier = c => c.MaxAge = now;
+                void cacheContextModifier(SourceCacheContext c) => c.MaxAge = now;
```

Comment:
any specific reason for changing from Action to this format? this is less readable in my opinion.

Reply:
Local functions are better than lambda expressions, as described [here](https://github.com/dotnet/roslyn/issues/3911).